packages/neuron-wallet/src/models/chain/script.ts

```diff
@@ -1,56 +1,8 @@
-import { scriptToHash } from '@nervosnetwork/ckb-sdk-utils'
+import { Script } from '@ckb-lumos/base'
 import HexUtils from '../../utils/hex'
-import TypeChecker from '../../utils/type-checker'
 
-export enum ScriptHashType {
-  Data = 'data',
-  Type = 'type',
-  Data1 = 'data1',
-}
-
-export default class Script {
-  public codeHash: string
-  public args: string
-  public hashType: ScriptHashType
-
-  constructor(codeHash: string, args: string, hashType: ScriptHashType) {
-    this.args = args
-    this.codeHash = codeHash
-    this.hashType = hashType
-
-    TypeChecker.hashChecker(this.codeHash)
-  }
-
-  public static fromObject({
-    codeHash,
-    args,
-    hashType,
-  }: {
-    codeHash: string
-    args: string
-    hashType: ScriptHashType
-  }): Script {
-    return new Script(codeHash, args, hashType)
-  }
-
-  public computeHash(): string {
-    const hash: string = scriptToHash(this.toSDK())
-    return HexUtils.addPrefix(hash)
-  }
-
-  public calculateBytesize(): number {
-    return 1 + HexUtils.byteLength(this.codeHash) + HexUtils.byteLength(this.args)
```

Comment:
Tried to replace with `blockchain.Script.pack(script).bytelength`, but that way the molecule encoding introduces 20 bytes of header. 

Not sure why we need this logic here but just keep it, if this logic is used to calculate tx size, then it can be refactored to `blockchain.Script.pack` in the future.

Reply:
This is not used to calculate tx size, it is used to calculate how many capacity needs to hold the cell.